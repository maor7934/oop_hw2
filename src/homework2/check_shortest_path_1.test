CreateNode n1 1
CreateNode n2 1
CreateNode n3 1
CreateNode n4 1
CreateNode n5 5
CreateNode n6 6
CreateNode n7 20
CreateGraph A
AddNode A n1
AddNode A n2
AddNode A n3
AddNode A n4
AddNode A n5
AddNode A n6
AddNode A n7
AddEdge A n1 n2
AddEdge A n2 n3
AddEdge A n3 n4
AddEdge A n4 n7
AddEdge A n1 n5
AddEdge A n1 n6
AddEdge A n5 n7
AddEdge A n6 n7
FindPath A n1 -> n7
