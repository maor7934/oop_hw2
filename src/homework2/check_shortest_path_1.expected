created node n1 with cost 1
created node n2 with cost 1
created node n3 with cost 1
created node n4 with cost 1
created node n5 with cost 5
created node n6 with cost 6
created node n7 with cost 20
created graph A
added node n1 to A
added node n2 to A
added node n3 to A
added node n4 to A
added node n5 to A
added node n6 to A
added node n7 to A
added edge from n1 to n2 in A
added edge from n2 to n3 in A
added edge from n3 to n4 in A
added edge from n4 to n7 in A
added edge from n1 to n5 in A
added edge from n1 to n6 in A
added edge from n5 to n7 in A
added edge from n6 to n7 in A
shortest path in A: n1 n2 n3 n4 n7
